hazelcast/src/main/java/com/hazelcast/cp/internal/MetadataRaftGroupManager.java

```diff
@@ -663,13 +669,17 @@ private void sendDestroyRaftNodeOps(CPGroupInfo group) {
         Operation op = new DestroyRaftNodesOp(Collections.singleton(group.id()));
         for (RaftEndpoint endpoint : group.members())  {
             if (endpoint.equals(localEndpoint)) {
-                raftService.destroyRaftNode(group.id());
+                destroyRaftNodeAsync(group.id());
             } else {
                 operationService.send(op, activeMembersMap.get(endpoint.getUuid()).getAddress());
             }
         }
     }
 
+    private void destroyRaftNodeAsync(CPGroupId groupId) {
+        nodeEngine.getExecutionService().execute(ASYNC_EXECUTOR, () -> raftService.destroyRaftNode(groupId));
```

Comment:
I think it would be better to use a new CP subsystem specific executor, instead of `ASYNC_EXECUTOR`.

Reply:
👍 will do